test/e2e/windows/utils.go

```diff
@@ -44,3 +49,20 @@ func waitForDeployment(getDeploymentFunc func() (*appsv1.Deployment, error), int
 		return util.DeploymentComplete(deployment, &deployment.Status), nil
 	})
 }
+
+// gets the container runtime and version for a node
+func getNodeContainerRuntimeAndVersion(n v1.Node) (string, semver.Version, error) {
+	containerRuntimeVersionString := n.Status.NodeInfo.DeepCopy().ContainerRuntimeVersion
+	parts := strings.Split(containerRuntimeVersionString, "://")
+
+	if len(parts) != 2 {
+		return "", semver.Version{}, fmt.Errorf("Could not get container runtime and version from '%s'", containerRuntimeVersionString)
```

Comment:
Idioms usually to start errors in lowercase

Reply:
updated